app/components/UI/FiatOnRampAggregator/components/TransactionDetails.tsx

```diff
@@ -167,67 +203,88 @@ const TransactionDetails: React.FC<Props> = ({ order }: Props) => {
 					<ListItem.Content style={styles.listItems}>
 						<ListItem.Body>
 							<Text black small>
-								Payment Method
+								{strings('fiat_on_ramp_aggregator.transaction.payment_method')}
 							</Text>
 						</ListItem.Body>
 						<ListItem.Amount>
 							<Text small bold primary>
-								{order.data.paymentMethod.name}
+								{order.paymentMethod?.name || data.paymentMethod?.name}
```

Comment:
Order does not have `paymentMethod`, we should rely on `data.paymentMethod.name`